examples/polymer/README.md

```diff
@@ -42,3 +41,11 @@ Polymer and its polyfills are intended to work in the latest version of [evergre
      `python -m SimpleHTTPServer`
 
 1. Browse to the server root
+
+## Making updates
+
+If you want to make a change to any of the elements in elements/elements.html, you'll need to install `polybuild` (Polymer's build tool) and re-vulcanize elements.build.html.
+
+1. Run `npm install`
+1. Make a change
```

Comment:
I was following the existing readme examples, but github will render a sequence: https://github.com/tastejs/todomvc/tree/gh-pages/examples/polymer#running-this-sample.